manifests/m/Microsoft/AzureCLI/2.53.1/Microsoft.AzureCLI.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.53.1
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.53.1.msi
```

Comment:
Was it intentional to remove the x86 installer? This URL is for the x86 (32-bit) installer. The x64 (64-bit) URL is:

https://azcliprod.azureedge.net/msi/azure-cli-2.53.1-x64.msi